### PR TITLE
Specify attribute 8 for async request status query

### PR DESCRIPTION
**Pull request by hansl**

### CODE REVIEW

File: attributes/network/8_async.adoc [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+===B1=== -->[ later] "unknown"
+
+@enduml
+....
+
+== Errors
+An error might be returned synchronously if the request is detected to be invalid (e.g. the signature doesn't match).
+
+If an error happens during processing, the resulting envelope should be cached the same as if the request was successful.
```
Comment (fmorency) [inline attributes/network/8_async.adoc:72]: If I understand correctly, the request will have the result "done" but with an error code inside the result payload?

**Review by fmorency — APPROVED** [review]
The content LGTM.

File: attributes/network/8_async.adoc (hunk 1 above)
Comment (hansl) [reply attributes/network/8_async.adoc:72]: Yes.

**Comment by hansl** [comment]
I'll do a full round of documentation to revise English later.

**Pull request merged**